frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -151,14 +163,7 @@ export class StorageClassDropdownInner extends React.Component<
       <>
         {loaded && itemsAvailableToShow && (
           <div>
-            <label
-              className={
-                this.props.hideClassName
```

Comment:
If hideClassName is a string that is used to pass classes. Why not pass just pass co-required in hideClassName prop?

Over here[1] just add the hideClassName prop and pass co-required and it should work.
Example:
```
                <StorageClassDropdown
                  onChange={handleStorageClass}
                  filter={onlyPvcSCs}
                  id="restore-storage-class"
                  hideClassName="co-required"
                  required
                />
```

[1]. https://github.com/openshift/console/pull/8055/files#diff-34a7d40d57e261c59626bc2fc32c3a2c0450366c6e0101651a49aac7591e7e3dR183

Reply:
name for the prop does not make sense though.... it is being used in select-capacity steps in the wizard but does not point to any css. Still not sure if thats what it is intended to do....

Reply:
as the intention of the prop is unclear and the PR also aims to fix the required props for the dropdown. i am keeping the current implementation instead of using hideClassName to add the css.